app/views/users/new.html.erb

```diff
@@ -1,3 +1,18 @@
 <% provide :title, t(".signup") %>
 <h1><%= t ".signup" %></h1>
-<p><%= t ".description" %>.</p>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for @user, url: signup_path do |f| %>
```

Comment:
In cho anh param nếu dùng form_for để đẩy lên và khi dùng form_tag để đẩy lên. Thử đổi form_tag để đăng ký user đi em

Reply:
+ Khi sử dụng form_for
![image](https://user-images.githubusercontent.com/67667699/87905611-34018080-ca8b-11ea-8e33-aea12903de4a.png)

+ Khi sử dụng form_tag
![image](https://user-images.githubusercontent.com/67667699/87905505-e71daa00-ca8a-11ea-8398-cf9116b6b875.png)

+ form_for được sử dụng khi muốn gửi  đối tượng lên server
+ form_tag được sử dụng khi không có đối tượng muốn gửi lên server

Reply:
![image](https://user-images.githubusercontent.com/67667699/88001245-d708d780-cb29-11ea-9ca1-44353fe09f2f.png)
em sử dụng form tag gửi dữ liệu lên và lưu vào db thành công rồi ạ